zeppelin/new_converter.py

```diff
@@ -0,0 +1,98 @@
+import re
+import os
+import base64
+from .converter import ZeppelinConverter
+
+
+class NewConverter(ZeppelinConverter):
+    """NewConverter class.
```

Comment:
Please remove the first line as it's meaningless. You can you the rest for the short description.